make.go

```diff
@@ -46,26 +39,24 @@ var (
 	// The project root where this file is located
 	projectRoot string
 	//
-	buildGoPath    string
 	buildOutputDir string
 )
 
 func usage() {
 	fmt.Println("Usage: go run make.go [command]")
 	fmt.Println("    all")
 	fmt.Println("    test")
-	fmt.Println("    setup\n")
 	os.Exit(1)
 }
 
 func main() {
+	_ = os.Setenv("GOFLAGS", "-mod=vendor")
```

Comment:
This didn't seem to take when I ran it. We might have to add `-mod=vendor` to build commands below